twentynineteen-blocks/block-templates/index.html

```diff
@@ -3,7 +3,9 @@
 <!-- /wp:group -->
 
 <!-- wp:group {"align":"full","className":"site-content"} -->
-<div class="wp-block-group alignfull site-content"><div class="wp-block-group__inner-container"><!-- wp:post-title /--><!-- wp:post-content /--></div></div>
+<div class="wp-block-group alignfull site-content"><div class="wp-block-group__inner-container">
+    <!-- wp:latest-posts {"postsToShow":100,"displayPostContent":true,"displayPostDate":true} /-->
```

Comment:
I'm showing 100 posts now because pagination is not supported yet.

Reply:
I'm also showing the excerpt, post date and title. Ideally, this would use a "Query" block where you could decide to show post comment count... (but the query block is not implemented yet).

Reply:
Definitely should be a Query since we need to read from the query context. It's also why I don't think adding pagination support to latest posts is a great idea: https://github.com/WordPress/gutenberg/pull/19969#issuecomment-586613632